ani-cli

```diff
@@ -426,14 +461,21 @@ case "$search" in
             [ -z "$query" ] && exit 1
         fi
         query=$(printf "%s" "$query" | sed "s| |+|g")
-        anime_list=$(search_anime "$query")
-        [ -z "$anime_list" ] && die "No results found!"
-        [ "$index" -eq "$index" ] 2>/dev/null && result=$(printf "%s" "$anime_list" | sed -n "${index}p")
-        [ -z "$index" ] && result=$(printf "%s" "$anime_list" | nl -w 2 | sed 's/^[[:space:]]//' | nth "Select anime: ")
-        [ -z "$result" ] && exit 1
-        title=$(printf "%s" "$result" | cut -f2)
+        if [ "$thumbnails" = "1" ]; then
+            search_anime "$query"
+            image_preview
+            title=$(printf "%s" "$choice" | sed -nE "s/(.*) \([0-9]* episodes\)  .*/\1 /p")
+            id=$(printf "%s" "$choice" | sed -nE "s/.* \([0-9]* episodes\)  (.*)\.jpg/\1/p")
+        else
+            anime_list=$(search_anime "$query")
+            [ -z "$anime_list" ] && die "No results found!"
+            [ "$index" -eq "$index" ] 2>/dev/null && result=$(printf "%s" "$anime_list" | sed -n "${index}p")
+            [ -z "$index" ] && result=$(printf "%s" "$anime_list" | nl -w 2 | sed 's/^[[:space:]]//' | nth "Select anime: ")
+            [ -z "$result" ] && exit 1
+            title=$(printf "%s" "$result" | cut -f2)
+            id=$(printf "%s" "$result" | cut -f1)
+        fi
```

Comment:
Due to placement of the code addition, I'm semi-confident that thumbnails will not work with history search